dapp/src/components/buySell/DisclaimerTooltip.js

```diff
@@ -24,15 +25,15 @@ const DisclaimerTooltip = ({
         {children || (
           <a className="ml-2" onClick={handleClick}>
             <img
```

Comment:
This help icon is causing some vertical alignment problems. Let's make the headings ("OUSD Balance" and "Trailing APY") line up, the big numbers, etc.

Reply:
Done